build/optaplanner-distribution-internal/pom.xml

```diff
@@ -50,7 +50,7 @@
     <!-- Binaries and sources -->
     <dependency>
       <groupId>org.optaplanner</groupId>
-      <artifactId>optaplanner-core</artifactId>
+      <artifactId>optaplanner-solver</artifactId>
```

Comment:
The rest of this file does import all pieces one by one (despite benchmark depending on persistence-xstream).
Shouldn't this list just specify all jars (core, score-* and solver)?